articles/virtual-desktop/publish-sandbox.md

```diff
@@ -78,16 +60,31 @@ Next, prepare the VM VHD for Azure and upload the resulting VHD disk to Azure. T
 Once you've uploaded the VHD to Azure, create a host pool that's based on this new image by following the instructions in the [Create a host pool by using the Azure Marketplace](create-host-pools-azure-marketplace.md) tutorial.
 
 ## Publish Windows Sandbox on your host pool
-
-To publish Windows Sandbox to your host pool, go to the [Manage app groups](manage-app-groups.md) section:
+### Option 1: Using the Azure Portal
+To publish Windows Sandbox to your host pool, log in to the portal and go to the [Manage app groups](manage-app-groups.md) section:
 
 1. Under "Application source" select "File Path"
 2. Under "Application path" enter "C:\windows\system32\WindowsSandbox.exe"
 3. Enter "Windows Sandbox" in "Application Name"
 
-That's it! Leave the rest of the options defaults. You should now have Windows Sandbox published for your users.
+### Option 2: Using Powershell
+1.  Open Windows Powershell on your local machine:
+2.  Log in to your Azure account using the following powershell command:
+```cmd
+az login
+
+Set-AzContext -Tenant [Workspace Tenant Id] -Subscription [Workspace Subscription Id]
+```
+3.  Creat a Sandbox Remote App using the following powershell command:
+```cmd
+New-AzWvdApplication -ResourceGroupName [Resource Group Name] -GroupName [Application Group Name] -FilePath 'C:\windows\system32\WindowsSandbox.exe' -IconIndex 0 -IconPath 'C:\windows\system32\WindowsSandbox.exe' -CommandLineSetting 'Allow' -ShowInPortal:$true -SubscriptionId [Workspace Subscription Id]
+```
+>[!NOTE]
+>This will prompt you for an app Name
+
+That's it! Leave the rest of the options defaults. You should now have Windows Sandbox published for your users. For reference your Remote Desktop app should now look like this:
 
-[TBD: Insert a screenshot here]
+![image](https://user-images.githubusercontent.com/106349886/170580444-71f221d8-348c-4124-8e88-94c0275b0f4c.png)
```

Comment:
Please use an image saved directly to the media folder in the virtual-docs docset on our local repo. External links aren't vetted by Microsoft, may cause legal issues if they involve branding from other companies, and can break.